plugins/inputs/postgresql/connect.go

```diff
@@ -79,21 +75,3 @@ func ParseURL(uri string) (string, error) {
 	sort.Strings(kvs) // Makes testing easier (not a performance concern)
 	return strings.Join(kvs, " "), nil
 }
-
-func Connect(address string) (*sql.DB, error) {
```

Comment:
I think this is used by the `postgresql_extensible` input, can you flip through that plugin and make sure its working and connecting in the correct way?